esmvalcore/config-developer.yml

```diff
@@ -21,121 +21,12 @@
 CMIP6:
   input_dir:
     default: '/'
-    BADC: '[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
+    BADC: '[activity]/[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
     DKRZ: '[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
```

Comment:
DKRZ follows the same layout (which is, I believe, just the esgf layout), so you can also add `[activity]/` in this line.

Note that both changes require a change to the rootpath in people's `config-user.yml`. Should we advertise this somehow? Maybe prominently in a changelog or release notes or something?